### PR TITLE
Made the controls be correct to the angle

### PR DESCRIPTION
**Pull request by bjornenalfa**
With these changes the control feeling has to be checked again and probably reconfigured.

### CODE REVIEW

File: src/game/entities/controllers/PlayerController.js [hunk 1]
```diff
@@ -28,24 +32,24 @@ class PlayerController extends EntityController {
...
+        let length = MAX_ANGLE;
+        const sqlength = beta * beta + gamma * gamma;
+        if (sqlength > MAX_ANGLE_SQ) {
+          length = Math.sqrt(sqlength);
+        }
+        beta /= length;
+        gamma /= length;
 
-        const length = Math.sqrt(beta * beta + gamma * gamma);
+        /* length = Math.sqrt(beta * beta + gamma * gamma);
```
Comment (joelnir) [inline src/game/entities/controllers/PlayerController.js]: Is there a point to keeping these comments? Seems like this is just the old implementation

Comment (bjornenalfa) [reply src/game/entities/controllers/PlayerController.js]: Comments should be removed, but they have been left to help with experimentation. The code in this specific comment had the unintended consequence of maximizing movement speed at a much lower angle than intended. Maybe this behaviour makes it feel better?

Comment (bjornenalfa) [reply src/game/entities/controllers/PlayerController.js]: Comments should be removed, but they have been left to help with experimentation. The code in this specific comment had the unintended consequence of maximizing movement speed at a much lower angle than intended. Maybe this behaviour makes it feel better?

Comment (joelnir) [reply src/game/entities/controllers/PlayerController.js]: Ok, so we'll just leave it for now?

Comment (bjornenalfa) [reply src/game/entities/controllers/PlayerController.js]: Yes, the system needs testing and probably tweaking. After that the code can be cleaned and merged into master.

**Comment by joelnir** [comment]
Big angles seems to give no or very small movement changes for me. Don't know if this is intended, but in the current state it is quite cumbersome to control the player

**Pull request merged**